model_analyzer/config/input/config_command_profile.py

```diff
@@ -574,7 +572,7 @@ def _add_triton_configs(self):
         self._add_config(
             ConfigField(
                 'triton_install_path',
-                field_type=ConfigPrimitive(str, validator=file_path_validator),
+                field_type=ConfigPrimitive(str),
```

Comment:
Remove the binary formatter for all launch modes.  Later, the `triton_install_path` path will be validated for `c_api` only.